src/Tickets/Seating/Admin/Ajax.php

```diff
@@ -59,19 +79,62 @@ class Ajax extends Controller_Contract {
 	 */
 	private Seat_Types $seat_types;
 
+	/**
+	 * A reference to the Sessions table object.
+	 *
+	 * @since TBD
+	 *
+	 * @var Sessions
+	 */
+	private Sessions $sessions;
+
+	/**
+	 * A reference to the Reservations service object.
+	 *
+	 * @since TBD
+	 *
+	 * @var Reservations
+	 */
+	private Reservations $reservations;
+
 	/**
 	 * Ajax constructor.
 	 *
 	 * @since TBD
 	 *
-	 * @param Container  $container  A reference to the DI container object.
-	 * @param Seat_Types $seat_types A reference to the Seat Types service object.
+	 * @param Container    $container  A reference to the DI container object.
+	 * @param Seat_Types   $seat_types A reference to the Seat Types service object.
+	 * @param Sessions     $sessions    A reference to the Sessions table object.
+	 * @param Reservations $reservations A reference to the Reservations service object.
 	 */
-	public function __construct( Container $container, Seat_Types $seat_types ) {
+	public function __construct(
+		Container $container,
+		Seat_Types $seat_types,
```

Comment:
How about we include the [`Service`](https://github.com/the-events-calendar/event-tickets/blob/92789c48eedfb10035d57209def09d6b1cd74ff8/src/Tickets/Seating/Service/Service.php#L91) instead? In that case we will have all of `seat_type`, `layouts` and `maps` from the Service.

Reply:
Right now the class does not require all those; just Seat Types. 
I would not pull something that is not being used yet.
If it turns out it's needed, I would change.